dependency-graph/dependency-review.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Dependency Review'
```

Comment:
Typically we suggest few things as comments at top of the workflow file:
- What is the value provided by this workflow
- Any pre-reqs whicih must be met for the workflow to have successful runs - think some assumptions, secrets, external entities which may be needed by Action
- Any doc/support link where user can reach out to read more

Reply:
The intent essentially is that user feel comfortable with the workflow. And understands enough to modify it for their use case if needed.

Reply:
cc @courtneycl @saritai for docs links 🙇

Reply:
Doc will live here: https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement

Reply:
I'll update the comment now